docs/guides/Getting-Started/fulfillment.md

```diff
@@ -124,77 +120,71 @@ Once you have your Shipping strategy set up, you will need to manage the items a
+* [POST - Acknowledgment reservation](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/inventory/reservations/-reservationId-/acknowledge)
+* [POST - Cancel reservation](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/inventory/reservations/-reservationId-/cancel)
 
 ### Scheduled delivery
 
 [Scheduled delivery](https://help.vtex.com/en/tutorial/scheduled-delivery--22g3HAVCGLFiU7xugShOBi) allows you to set delivery time frames for shoppers to choose the day and time to receive the order.
 
-* [POST - Add blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/logistics/pvt/configuration/carriers/-carrierId-/adddayofweekblocked)
-* [GET - Retrieve blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#get-/logistics/pvt/configuration/carriers/-carrierId-/getdayofweekblocked)
-* [POST - Remove blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/logistics/pvt/configuration/carriers/-carrierId-/removedayofweekblocked)
-* [GET - Search capacity reservations in time range](https://developers.vtex.com/docs/api-reference/logistics-api#get-/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames)
-* [GET - Get capacity reservation usage by window](https://developers.vtex.com/docs/api-reference/logistics-api#get-/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames/-windowDay-F-windowStartTime-T-windowEndTime-)
+* [POST - Add blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/carriers/-carrierId-/adddayofweekblocked)
+* [GET - Retrieve blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics/pvt/configuration/carriers/-carrierId-/getdayofweekblocked)
+* [POST - Remove blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/carriers/-carrierId-/removedayofweekblocked)
+* [GET - Search capacity reservations in time range](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames)
+* [GET - Get capacity reservation usage by window](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames/-windowDay-F-windowStartTime-T-windowEndTime-)
 
 ### Holidays
 
 The [Holidays](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) feature allows you to configure days that should not be considered valid for delivery so that VTEX platform considers those days when calculating the shipping estimated time.
```

Comment:
```suggestion
The [Holidays](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) feature allows you to configure days that should not be considered valid for delivery, so that the VTEX platform adds those days when calculating the shipping estimated time.
```